### PR TITLE
Cleanup user guide sections d-f

### PR DESCRIPTION
**Pull request by kprosise**
Three pages were cleaned up to meet style guidelines, and updated where needed.

This included removing dead internal references/links and simplifying wording. Semantic newlines were added.
Note that not all information was verified for being up to date, as the focus was primarily on style.

The issues stated in FFTK 2174 have been addressed except for item 3, which is out of our control.

QA Steps: vale linter, linkcheck, viewed rendered HTML.

This commit addresses FFTK 2527
This commit addresses FFTK 2174
This commit applies to FFTK 988

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

File: source/user-guide/el2g.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Integrating NXP EdgeLock 2GO
 ============================
 
-EdgeLock® 2GO is an add-on service from NXP® that is integrated into FoundriesFactory as an option to simplify the securing of devices during manufacturing.
+EdgeLock® 2GO is an add-on service from NXP® ,integrated into FoundriesFactory® to simplify the securing of devices during manufacturing.
```
Comment (angolini) [inline source/user-guide/el2g.rst:6]:
```suggestion
EdgeLock® 2GO is an add-on service from NXP®, integrated into FoundriesFactory® to simplify the securing of devices during manufacturing.
```

File: source/user-guide/el2g.rst [hunk 2]
```diff
@@ -245,4 +246,7 @@ API documentation links:
  * `Developer Guide <https://cdn.foundries.io/el2go/AN12642_EdgeLock_2GO_Developer_Guide_for_Foundries.io_users.pdf>`_
  * `OpenAPI Swagger <https://cdn.foundries.io/el2go/el2go-managed-api-gateway-api-58.45.0.yaml>`_
 
-The default FoundriesFactory EdgeLock 2GO implementation provides a free of charge evaluation for 30 days. Once enabled for commercial use the standard package limits usage to 50,000 devices per subscription year and 2x key pairs and 2x X.509 certificates per device - i.e. the FoundriesFactory key pair and certificate, and one additional set for authentication to a third party service such as AWS. If you require additional devices, or more key pairs per device, please contact us.
+The default FoundriesFactory EdgeLock 2GO implementation provides a free of charge evaluation for 30 days.
+Once enabled for commercial use, the standard package limits usage to 50,000 devices per subscription year and 2x key pairs and 2x X.509 certificates per device.
+This covers the FoundriesFactory key pair and certificate, and one additional set for authentication to a third party service such as AWS.
```
Comment (angolini) [inline source/user-guide/el2g.rst:251]:
```suggestion
This covers the FoundriesFactory key pair and certificate and one additional set for authentication to a third-party service such as AWS.
```

File: source/user-guide/fioctl/index.rst [hunk 3]
```diff
@@ -3,27 +3,27 @@
...
 .. _ug-fioctl-enable-apps:
 
 Enabling/Disabling Apps
 -----------------------
 
-By default all apps defined in :term:`containers.git` for any given tag are
-enabled. To change this behaviour, a list of allowed apps can be given **per
-device**, enabling only those apps that are in a comma separated list.
+All apps defined in :term:`containers.git` for any given tag are enabled by default.
+To change this behavior, allowed apps can be set **per device**, enabling only those in a comma separated list.
```
Comment (angolini) [inline source/user-guide/fioctl/index.rst:19]:
```suggestion
To change this behavior, allowed apps can be set **per device**, enabling only those in a comma-separated list.
```

File: source/user-guide/fioctl/index.rst [hunk 4]
```diff
@@ -217,8 +210,7 @@ View Targets
 
 List devices
   ``fioctl devices list -f <factory>``
-    Lists the devices that have connected to a Factory, along with associated
-    metadata, such as device name, status, Target and enabled apps.
+    Lists the devices that have connected to a Factory, along with metadata, such as device name, status, Target, and enabled apps.
```
Comment (angolini) [inline source/user-guide/fioctl/index.rst:213]:
```suggestion
    Lists the devices connected to a Factory and metadata, such as device name, status, Target, and enabled apps.
```

**Review by angolini — APPROVED** [review]

**Comment by doanac** [comment]
Docs for bbae516095f6aa055180046090d8ea9363ca0710 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2327/docs/artifacts/html/index.html

**Pull request merged**